control/velocity_controller/cfg/VelocityController.cfg

```diff
@@ -1,69 +0,0 @@
-#! /usr/bin/env python
-
-PACKAGE = 'velocity_controller'
-
-from dynamic_reconfigure.parameter_generator_catkin import *
```

Comment:
I suppuose no callback are used so here you don't implement any dynamic reconfigure capabilities? Just checking because `velocity controller` sounds like you might want to tune in the car. I'm porting the MPC_follower and that needs

Reply:
Sorry, I should add some comments on this point. I modified the top comment. Please check it.